### DISCUSSION TITLE
part-ii/2021-0/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#112] [Question comments]
Part II 2021 Algebraic Geometry: Paper 1, Section II, I 

https://questions.tripos.org/part-ii/2021-0/

**Comment by eha1234** [comment]
Bookwork for the first few bits. 
Then $V(X^2+Y^2-1, Y+iZ) \cup V(X^2+Y^2-1, Y-iZ)$. For the last bit, this was on Example Sheet 1 of 2021/2022, which gives a hint on how to proceed.
